### DISCUSSION TITLE
Connection is refused but internet is connecting

### DISCUSSION THREAD

**Discussion by tri-hariyadi** [chris1111/Wireless-USB-Big-Sur-Adapter#83] [Q&A]
I can't connect to the internet even though my computer is still connected to the mobile hotspot with a good connection. this happens when i use mac os big sur version 11.2.1 with usb dongle tp-link TL-WN823N v3

**Comment by mhristev** [comment]
Same here, did you manage to fix it?

**Reply by tri-hariyadi** [reply]
yes, I managed to solve this problem by replacing my usb wifi adapter with an tp-link Archer T2U Nano AC600 type and upgrading driver version 11, now the driver is working fine and no more problems.
